crates/re_data_ui/src/lib.rs

```diff
@@ -62,6 +64,10 @@ pub trait DataUi {
         query: &re_data_store::LatestAtQuery,
         db: &re_entity_db::EntityDb,
     );
+
+    fn data_ui_default(&self, ctx: &ViewerContext<'_>, ui: &mut egui::Ui, ui_layout: UiLayout) {
```

Comment:
misleading name (saying this as someone who displays data from many stores ;))! how about
```suggestion
    fn data_ui_recording(&self, ctx: &ViewerContext<'_>, ui: &mut egui::Ui, ui_layout: UiLayout) {
```
(I'd like to capture the fact that it's the current query, but that's how we always do it :shrug:)

Reply:
(sidenote: the fact that we pass through query & db is a source of headache because it implies that DataUi does additional queries sometimes, but let's take that later)